gap/attributes/rms-translat.gd

```diff
@@ -0,0 +1,38 @@
+############################################################################
+##
+# W  rms-translat.gd
+# Y  Copyright (C) 2015-22                                       Finn Smith
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+# The declarations in this file are purposefully undocumented.
```

Comment:
They are internal, which I've noted in the file, but should I also preface them with SEMIGROUPS? I was avoiding doing so just because of how annoying that looks when they are used as filters.